interface/modules/custom_modules/oe-module-claimrev-connect/templates/navbar.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ *
+ * @author    Brad Sharp <brad.sharp@claimrev.com>
+ * @copyright Copyright (c) 2022 Brad Sharp <brad.sharp@claimrev.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+ ?>
+ <?php echo xlt(""); ?>
```

Comment:
above line not needed

Reply:
you found my copy paste line. forgot to delete that.